### ISSUE TITLE
[BUG] [Service] [getPattern] Week info("firstDayOfWeek" &"weekendOfRange") should also follow CLDR data

### ISSUE THREAD

**Issue by nannany00** [vmware/singleton#819]
**Describe the bug**
For Week info (**firsDayOfWeek** & "weekendOfRange") under **dates** pattern is hardcode for all locales, it should also follow CLDR data.

**To Reproduce**
1. Go to  getPatternByLanguageAndRegion API
2. Give the request data
```
language: en
region: AD
scope:dates
```
**Response:**
```
firstDayOfWeek: 0
```
**Expect Result:**
```
firstDayOfWeek: 1
```
**Expected behavior**
The pattern data of *firstDayOfWeek* should follow CLDR definition.

**Additional context**
CLDR data: https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/weekData.json

**Comment by nannany00** [comment]
Waiting for a new Singleton Pattern package to do testing.

**Comment by huihuiw01** [comment]
> Waiting for a new Singleton Pattern package to do testing.

I have asked @junw-33455432  to help publish a new pattern build last Wednesday, he has started working on this last Friday.
@junw-33455432 Could you update the build status here once build is ready? Then @nannany00 can start testing.

**Comment by lyiyu66** [comment]
The issue is fixed with commit 5414eb6939a4c4414f65cf091bc67e4737e38f07 and 0.5.2 pattern package.

![image](https://user-images.githubusercontent.com/51813046/103497548-d0261500-4e7c-11eb-9534-4744c7f7105e.png)
